content/articles/statistical-hypothesis-testing-python-implementation/index.md

```diff
@@ -0,0 +1,907 @@
+**"She has fibroid"** 
+
+while our alternate hypothesis will be
+**"She doesn't have fibroid"**
+
+```
+H0 = "She has fibroid"
+H1 = "She has no fibroid" (Maybe she's pregnant)
+```
+
+In this case, after carrying out hypothesis testing on her condition, will it be safe to make the mistake of saying or assuming
+**"She has no fibroid",** 
+when she actually has fibroid? 
+or will it be costlier to make the assumption of  
+**"She has fibroid",**
+When she's actually miraculously pregnant like Hannah?.
+
+There is someone's life involved here so you have to think it through.
+
+### Confidence of Null Hypothesis (Probability Value)
+
+Technically, the only way we can accept or reject our ground truth (null hypothesis) is after determining our *P-value* which is an abbreviation for probability value. It is the only thing that has the decisive power to accept or reject H0.
+
+***The smaller our P-values is the more delicate it is to accept our null hypothesis.***
+
+To calculate our P-value, there should be a certain level of significance to our ground truth or initial assumption, so a smaller P-value is telling us there are more reasons why we should accept our alternative hypothesis.
+
+P-value is within the range of 0 and 1, but the threshold value is always set at 0.05, it can also be called *alpha*, alpha is always set before the experiments in order to avoid bias. 
+
+Our alpha value is the probability of making a wrong decision when the ground truth(H0) is true. It is always calculated as 
+
+```
+alpha = 1 - confidence level
+```
+
+if the confidence level from what we've seen in our data is 95%, then 
+
+```
+alpha = 1 - 0.95
+```
+
+which is 0.05.
+
+### Why is 95% confidence level the most accustomed?
+
+Since we know that alpha level is the probability of encountering a Type I error(which is rejecting H0 despite being true), would it not make sense to make it as bitsy as possible. Imagine our alpha value being set to 10%, obviously, there is an increasing chance that we will make the mistake of rejecting the null hypothesis, but it must not be too small as well, if we set the alpha value as 1%, it will be too small that our decision will be reeking of bias. Hence, the standard has been 5%.
+
+### Statistical Techniques for Hypothesis Testing
+
+In this tutorial, I will be touching on three methods of carrying out statistical hypothesis testing, namely:
+
+1. Chi-Square Test
+2. T-Test
+3. ANOVA Test
+
+### Chi-Square Test
+
+This is used to perform testing on two categorical variables in our data population. It is focused on looking for an important relationship between two categorical variables.
+
+With Chi-Square, we accept the null hypothesis when variable A and variable B are not dependent, i.e. there is no relationship between them.
+We reject the null hypothesis(or accept the alternate hypothesis) when variable A depends on variable B, i.e. there is a relationship between them.
+
```

Comment:
Chi-Square tests are on the most-used end of statistical tests, so it is imperative that the reader has a good understanding of how a chi-square test work. Do explain them a bit more, and maybe add Section articles and resources.